src/commands/qmis.ts

```diff
@@ -88,6 +96,18 @@ export default class QMIS extends Command {
         await DELETE.qmis(id);
         this.log('QMI deletion successful.');
       } else { this.log('Typed response doesn\'t match QMI IP address, aborting deletion.'); }
+    } else if (flags.start || flags.stop) {
+      if (flags.keypath) throw new Error(`Cannot supply a --keypath when powering on a QMI.`);
+      if (!flags.id) throw new Error(`Must supply an --id when deleting a QMI.`);
+
+      const id = parsePositiveInteger(flags.id);
+      if (!id) { throw new Error(`Must supply a positive integer ID when powering on/off a QMI.`); }
+
+      try {
+        const qmiURLAction = flags.start ? 'start' : 'stop';
+        await POST.qmi(id, qmiURLAction);
+        this.log(`QMI successfully ${flags.start ? 'powered on' : 'powered off'}`);
```

Comment:
Add a trailing period to the sentence (assuming that's compatible with what we do elsewhere, can't remember)